docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -41,32 +41,11 @@ Refer to [Cloud provider regions](../release-notes#cloud-provider-regions) for a
 
 Yugabyte Cloud supports all the regions that have robust infrastructure and sufficient demand from customers. We are continuously improving region coverage, so if there are any regions you would like us to support, reach out to [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431).
 
-### What version of YugabyteDB does Yugabyte Cloud run on
-
-Yugabyte Cloud runs on the YugabyteDB 2.8 [stable release](../../releases/whats-new/stable-release/).
-
-To see the version you are running, connect in cloud shell using YSQL and run the following command:
-
-```sh
-yugabyte=> SELECT version();
-```
-
-```output
-                                                  version                                                   
-------------------------------------------------------------------------------------------------------------
- PostgreSQL 11.2-YB-2.8.0.0-b0 on x86_64-pc-linux-gnu, compiled by gcc (Homebrew gcc 5.5.0_4) 5.5.0, 64-bit
-(1 row)
-```
-
-### Can I test YugabyteDB locally?
-
-To test locally, [download](https://download.yugabyte.com) and install YugabyteDB on a local machine. Refer to [Quick Start](../../quick-start). For accurate comparison with cloud, be sure to download the version that is running on Yugabyte Cloud.
-
 ### What are the differences between free and standard clusters?
 
-Use the free cluster to get started with YugabyteDB. The free cluster is limited to a single node and 10GB of storage, and although not suitable for production workloads, the cluster includes enough resources to start exploring the core features available for developing applications with YugabyteDB. You can only have one free cluster.
+Use the free cluster to get started with YugabyteDB. The free cluster is limited to a single node and 10GB of storage, and although not suitable for production workloads, the cluster includes enough resources to start exploring the core features available for developing applications with YugabyteDB. Free clusters are provisioned with the Latest release version of YugabyteDB. You can only have one free cluster.
 
-Standard clusters can have unlimited nodes and storage and are suitable for production workloads. They also support horizontal and vertical scaling - nodes and storage can be added or removed to suit your production loads. Standard clusters also support VPC peering, and scheduled and manual backups.
+Standard clusters can have unlimited nodes and storage and are suitable for production workloads. They also support horizontal and vertical scaling - nodes and storage can be added or removed to suit your production loads. Standard clusters also support VPC peering, and scheduled and manual backups. Standard clusters are provisioned with the Stable release version of YugabyteDB.
```

Comment:
Worth linking to the release notes for Stable in the last sentence?

Reply:
similar comment to above; "cloud stable release" != OSS/platform stable release all of the time; it has its own definition.

Also FYI this is going to change within ~1 week in PROD (the change is making it's way up our environment chain to prod already), but paid tier clusters will get a choice of which release track to be pinned to during cluster creation (i.e. choice between edge, stable, or any custom tracks we may have added for a given account)